public/scripts/fetchResults.js

```diff
@@ -159,4 +188,4 @@ document.addEventListener('DOMContentLoaded', () => {
   }
 });
 
-export { hydrateSortOption, invokeSearchResults };
+export { hydrateSortOption, hydratePageResultsOption, invokeSearchResults };
```

Comment:
This is a very rare case if someone inspects and makes direct changes at the local storage. We can access the search results. Similar to this code https://github.com/DEFRA/ncea-frontend/blob/24dd98872658c756408ea5b1d1a2973da4c1e977/public/scripts/customScripts.js#L211